# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/runit/rockify.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

we need to export vars to be read inside init.d/system
currently we simulate sysv's stage 2
  Alejandro

--- ./doc/debian/1.orig	2003-10-29 18:28:27.000000000 +0000
+++ ./doc/debian/1	2003-11-11 02:43:07.000000000 +0000
@@ -1,10 +1,9 @@
 #!/bin/sh
 # system one time tasks
 
-PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
+export PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
 
-/etc/init.d/rcS
-/etc/init.d/rmnologin
+/etc/rc.d/init.d/system start
 
 touch /etc/runit/stopit
 chmod 0 /etc/runit/stopit
--- ./etc/debian/3.orig	2003-11-18 12:47:03.000000000 -0300
+++ ./etc/debian/3	2003-12-07 10:28:44.000000000 -0300
@@ -1,10 +1,10 @@
 #!/bin/sh
 exec 2>&1
 
-PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
+export PATH=/command:/sbin:/bin:/usr/sbin:/usr/bin
 
-LAST=0
-test -x /etc/runit/reboot && LAST=6
+export RUNLEVEL=0
+test -x /etc/runit/reboot && export RUNLEVEL=6
 
 echo 'Waiting for getties to stop...'
 svwaitdown -xk -t14 /var/service/getty-*
@@ -13,4 +13,4 @@
 svwaitdown -xk -t350 /var/service/*
 
 echo 'Shutdown...'
-/etc/init.d/rc $LAST
+/etc/rc.d/init.d/system stop
